src/main/java/io/getstream/chat/java/services/UserService.java

```diff
@@ -64,5 +64,6 @@
   Call<StreamResponseObject> unban(
       @NotNull @Query("target_user_id") String targetUserId,
       @Nullable @Query("type") String channelType,
-      @Nullable @Query("id") String channelId);
+      @Nullable @Query("id") String channelId,
+      @Nullable @Query("shadow") Boolean shadow);
```

Comment:
Seems like this param doesn't have any effect. It removes 2 types of bans. @ferhatelmas

Reply:
interesting. I couldn't find it in the REST docs, but Python/Ruby etc has it

Reply:
https://github.com/GetStream/stream-chat-ruby/blob/602802956c475c2fdfcd168aafff70edfb042d19/lib/stream-chat/client.rb#L164

Reply:
I see go sdk has it too 🤔. Alright, let's keep it for consistency

Reply:
and the tests succeed as well, so.. 🤔